d-sgov/doc-ofn-adresy-1.0.1-glosář.ttl

```diff
@@ -62,6 +62,15 @@ doc-ofn-adresy-1.0.1-pojem:vúsc
         <http://www.w3.org/2004/02/skos/core#prefLabel>
                 "vúsc"@cs .
 
+doc-ofn-adresy-1.0.1-pojem:momc
+        a       <http://www.w3.org/2004/02/skos/core#Concept> ;
+        <http://www.w3.org/2004/02/skos/core#broader>
+                <https://slovník.gov.cz/základní/pojem/vztah> , <https://slovník.gov.cz/základní/pojem/typ-vztahu> ;
+        <http://www.w3.org/2004/02/skos/core#inScheme>
+                doc-ofn-adresy-1.0.1:glosář ;
+        <http://www.w3.org/2004/02/skos/core#prefLabel>
+                "momc"@cs .
```

Comment:
VÚSC/MOMC ... - neměli bychom jako prefLabel používat zkratky. Ve SKOS na to máme skos:abbrLabel (nevím ale, zda modelio ho exportuje)

Reply:
v modeliu o tom nevím. jestli je to tak velký problém, tak přepíšu labely na celé názvy, ale bude to teda pekelně dlouhý :)

Reply:
nahrazeno v dalším commitu